exercises/concept/cars-assemble/.docs/instructions.md

```diff
@@ -20,7 +20,7 @@ CarsAssemble.productionRatePerHour(6)
 // => 1193.4
 ```
 
-Note that the value returned is a `double`.
+Note that the value returned is a `double`. You will need to take the success rate into account to get the correct result.
```

Comment:
Hey @tomasnorre thanks for raising this number and taking into account the number of breaking solutions!

I noticed the end of line 16 also talks about something that students to have to take into account. I suggest updating this task so that its all in one place. For example, instead of adding this sentence here, we could add to line 16 like:

```
Implement the `CarsAssemble.productionRatePerHour()` method to calculate the assembly line's production rate per hour, taking into account its current assembly line's speed _and_ success rate:
```

Reply:
I like your suggestion better than mine :)